bank/sql_bank/README.md

```diff
@@ -0,0 +1,37 @@
+# Bank example
+
+## Summary
+
+The bank example program continuously performs balance transfers between
+accounts using concurrent transactions.
+
+## Running
+
+Run against an existing cockroach node or cluster.
+
+#### Development node
+```
+# Build cockroach binary from https://github.com/cockroachdb/cockroach
+# Start it in dev mode (listens on localhost:26257)
+./cockroach start --dev
+
+# Build sql_bank example.
+# Start it with:
+./sql_bank --db-url=http://localhost:26257
```

Comment:
Since this always needs to be specified, let's not use a flag but instead use `flags.Arg(0)`.